modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -3673,6 +3673,8 @@ channels is derived automatically from src and code.
  */
 CV_EXPORTS_W void cvtColor( InputArray src, OutputArray dst, int code, int dstCn = 0 );
 
+CV_EXPORTS_W void cvtColorTwoPlane( InputArray src1, InputArray src2, OutputArray dst, int code, int dcn = 0 );
```

Comment:
> dcn

Why? RGB/RGBA codes are different, so we can properly detect "dcn" value like `cvtColor` does.

Reply:
Ah, right. How about the name? Should I just overload `cvtColor`? After thinking about it a bit more, that actually seems cleaner to me...

Reply:
OK, fixed. Method name still as-is.